File: TutorialGame/TutorialGame/Game/DebugUtility.cs
using System;
using SFML.Graphics;
using SFML.System;

namespace TutorialGame
{
    public static class DebugUtility
    {

        public const string CONSOLE_FONT_PATH = "./Fonts/ARIAL.TTF";

        public static Font consoleFont;
        public static void LoadContent()
        {
            consoleFont = new Font(CONSOLE_FONT_PATH);
        }

        public static void DrawPerformanceData(GameLoop gameLoop, Color fontColor)
        {
            if (consoleFont == null)
                return;

            string totalTimeElapsedStr = gameLoop.GameTime.TotalTimeElapsed.ToString("0.000");
            string deltaTimeStr = gameLoop.GameTime.DeltaTime.ToString("0.00000");
            float fps = 1f / gameLoop.GameTime.DeltaTime;
            string fpsStr = fps.ToString("0.00");

            Text textA = new Text(totalTimeElapsedStr, consoleFont, 14);
            textA.Position = new Vector2f(4f, 8f);
            textA.FillColor = fontColor;

            Text textB = new Text(deltaTimeStr, consoleFont, 14);
            textB.Position = new Vector2f(4f, 28f);
            textB.FillColor = fontColor;

            Text textC = new Text(fpsStr, consoleFont, 14);
            textC.Position = new Vector2f(4f, 48f);
            textC.FillColor = fontColor;

            gameLoop.Window.Draw(textA);
            gameLoop.Window.Draw(textB);
            gameLoop.Window.Draw(textC);

        }

    }
}

File: TutorialGameVB/TutorialGameVB/Game/DebugUtility.vb
Imports System
Imports SFML.Graphics
Imports SFML.System

Namespace TutorialGame

    Module DebugUtility

        Public Const CONSOLE_FONT_PATH As String = "./Fonts/ARIAL.TTF"

        Public consoleFont As Font

        Sub LoadContent()
            consoleFont = New Font(CONSOLE_FONT_PATH)
        End Sub

        Sub DrawPerformanceData(ByVal gameLoop As GameLoop, ByVal fontColor As Color)

            If consoleFont Is Nothing Then Return
            Dim totalTimeElapsedStr As String = gameLoop.GameTime.TotalTimeElapsed.ToString("0.000")
            Dim deltaTimeStr As String = gameLoop.GameTime.DeltaTime.ToString("0.00000")
            Dim fps As Single = 1.0F / gameLoop.GameTime.DeltaTime
            Dim fpsStr As String = fps.ToString("0.00")
            Dim textA As Text = New Text(totalTimeElapsedStr, consoleFont, 14)
            textA.Position = New Vector2f(4.0F, 8.0F)
            textA.FillColor = fontColor
            Dim textB As Text = New Text(deltaTimeStr, consoleFont, 14)
            textB.Position = New Vector2f(4.0F, 28.0F)
            textB.FillColor = fontColor
            Dim textC As Text = New Text(fpsStr, consoleFont, 14)
            textC.Position = New Vector2f(4.0F, 48.0F)
            textC.FillColor = fontColor
            gameLoop.Window.Draw(textA)
            gameLoop.Window.Draw(textB)
            gameLoop.Window.Draw(textC)

        End Sub

    End Module

End Namespace